base/linalg/arpack.jl

```diff
@@ -122,9 +122,9 @@ function eupd_wrapper(T, n::Integer, sym::Bool, cmplx::Bool, bmat::String,
     elseif which == "SR" || which == "SA"
         dmap = x->-real(x)
     elseif which == "LI"
-        dmap = imag
+        dmap = x -> imag.(x)
     elseif which == "SI"
-        dmap = x->-imag(x)
+        dmap = x -> (-).(imag.(x))
```

Comment:
Not that I am aware of. Exploiting fusion was the objective above. Are you suggesting that we should consider deprecating vectorized unary minus? Best!